src/http.zig

```diff
@@ -1506,6 +1506,7 @@ redirect_type: FetchRedirect = FetchRedirect.follow,
 redirect: []u8 = &.{},
 timeout: usize = 0,
 progress_node: ?*std.Progress.Node = null,
+progress_formatter: ?*const fn (size: usize) usize = null,
```

Comment:
is using a function pointer for this necessary? more specifically, do we actually use progress_node anywhere else where we want a different formatter.

i think we would want all uses of this to be formatted nicely. i think it would be Ok to always format to MB if there is a progress_node set.

Reply:
I think I will remove this and always format to a larger unit as you suggested